src/app/(secure)/report-types/select-object/page.tsx

```
@@ -1,6 +1,6 @@
 import SelectObjectForm from "@/components/report-type/select-object-form";
 
-export default function ReportTypeSelectionPage() {
-  console.log("Create report type page ------------------------");
-  return <SelectObjectForm reportTypeId="" ></SelectObjectForm>
+export default function ReportTypeSelectionPage({ params }: { params: any }) {
```

ESLint

[error] 'params' is defined but never used.

(@typescript-eslint/no-unused-vars)